README.md

```diff
@@ -172,7 +172,7 @@ Running `npm test` command will start [Mocha](https://mochajs.org/) tests via [K
 
 ## Bug Report
 
-If you find a bug, please report to us opening a new [Issues](https://github.com/naver/egjs-flicking/issues) on GitHub.
+If you find a bug, please report to us opening a new [Issues](https://github.com/naver/egjs-flicking/issues) on GitHub with tag `2.x`.
```

Comment:
@WoodNeck `label` is more proper than `tag`

Reply:
Oops, I'll fix that